components/03-modules/pager/Pager.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <nav class="pager">
+    <ul class="pager__list">
+      <li class="pager__item">
+        <alpaca-button 
+          icon="arrow-left" 
+          custom-class="pager__link pager__link--prev"
+          aria-label="Go to previous page"
+          :disabled="currentPage < 2"
+          @click="currentPage--"
+        />
+      </li>
+
+      <template v-if="showFirst">
+        <li class="pager__item">
+          <alpaca-button
+            custom-class="pager__link"
+            @click="cyrrentPage = 1"
+          >
+            1
+          </alpaca-button>
+        </li>
+        <li class="pager__item">
+          ...
+        </li>
+      </template>
+
+      <li
+        v-for="number in limitedPageNumbers"
+        :key="number"
+        :class="[
+          'pager__item',
+          { 'pager__item--current': currentPage === number }
+        ]"
+      >
+        <alpaca-button
+          custom-class="pager__link"
+          @click="currentPage = number"
+        >
+          {{ number }}
+        </alpaca-button>
+      </li>
+
+      <template v-if="showLast">
+        <li class="pager__item">
+          ...
+        </li>
+        <li class="pager__item">
+          <alpaca-button
+            custom-class="pager__link"
+            @click="currentPage = numberOfPages"
```

Comment:
Maybe we should create methods setCurrentPage ? :)

Reply:
IMO it's better to create a function that takes number as an argument and than sets this currentPage. I think, it could be more reusable with the future development in this component.

Reply:
updated